roles/ee_builder/vars/main.yml

```diff
@@ -3,8 +3,6 @@
 __ee_stream_images:
   upstream:
     base_image: quay.io/ansible/ansible-runner:latest
-    builder_image: quay.io/ansible/ansible-builder:latest
   downstream:
-    base_image: registry.redhat.io/ansible-automation-platform-23/ee-minimal-rhel8:latest
-    builder_image: registry.redhat.io/ansible-automation-platform-23/ansible-builder-rhel8:latest
+    base_image: registry.redhat.io/ansible-automation-platform-24/ee-minimal-rhel8:latest
```

Comment:
I think we should be using the RHEL9 image as the base instead of RHEL8 as the default. With as large of a change as this is I think it makes sense to also default to RHEL9 now instead of later.

Reply:
yes, already updated